build/ci/check_demo_pr_checks.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export passed_k8s_checks=true
+export passed_openshift_checks=true
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  is_summary_test_passed=false
+  export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    sleep 5
+    while [ "$(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i pending | wc -l)" -gt 0 ]; do
```

Comment:
this long `gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository` command repeats a lot here, please put it in a var